x/slashing/autocli.go

```diff
@@ -0,0 +1,48 @@
+package slashing
+
+import (
+	"strings"
+
+	autocliv1 "cosmossdk.io/api/cosmos/autocli/v1"
+	slashingv1beta "cosmossdk.io/api/cosmos/slashing/v1beta1"
+)
+
+func (am AppModule) AutoCLIOptions() *autocliv1.ModuleOptions {
+	return &autocliv1.ModuleOptions{
+		Query: &autocliv1.ServiceCommandDescriptor{
+			Service: slashingv1beta.Query_ServiceDesc.ServiceName,
+			RpcCommandOptions: []*autocliv1.RpcCommandOptions{
+				{
+					RpcMethod: "Params",
```

Comment:
The params are wrongly encoded:

``go
params:
  downtime_jail_duration: 600s
  min_signed_per_window: NTAwMDAwMDAwMDAwMDAwMDAw
  signed_blocks_window: "100"
  slash_fraction_double_sign: NTAwMDAwMDAwMDAwMDAwMDA=
  slash_fraction_downtime: MTAwMDAwMDAwMDAwMDAwMDA=
```